Python/sysmodule.c

```diff
@@ -2821,6 +2926,17 @@ _PySys_InitCore(PyThreadState *tstate, PyObject *sysdict)
         }
     }
 
+#ifdef __EMSCRIPTEN__
+    if (EmscriptenInfoType.tp_name == NULL) {
+        if (_PyStructSequence_InitType(&EmscriptenInfoType,
```

Comment:
Would it be possible to use PyStructSequence_NewType() instead? See Modules/_cursesmodule.c which creates a structseq type, create an instance, and then delete the type. It would avoid to add yet another static type to Python: see https://peps.python.org/pep-0687/ and https://peps.python.org/pep-0630/

Reply:
done